src/ssl.cxx

```diff
@@ -31,8 +32,15 @@
 #ifdef HAVE_LIBSSL
 static std::unique_ptr<SSL_CTX, std::function<void(SSL_CTX*)>> ssl;
 
+# if OPENSSL_VERSION_NUMBER < 0x10000000L
 static void key_progress_cb(int p, int n, void* arg)
+# else
+static int key_progress_cb(int p, int n, BN_GENCB *cb)
```

Comment:
Please respect the original code style, in particular use of spaces etc.

Reply:
Indeed, sorry.
Do you mean the `# else` bit (should I use a tab? or no space at all?), or the `BN_GENCB *cb` one? Or both?

Reply:
Both/all of them. I'm not sure what are the conventions in the surrounding code but I think no space would be better here.

Reply:
There are tabs used in `#	include` statements inside a `#if` block at the top of the file, but that didn't feel very readable inside a code block. I'll go with no space as you suggest.